test/src/main/java/org/apache/accumulo/test/TableOperationsIT.java

```diff
@@ -202,6 +204,60 @@ public void createTable() throws TableExistsException, AccumuloException,
     accumuloClient.tableOperations().delete(tableName);
   }
 
+  @Test
+  public void testDefendAgainstThreadsCreateSameTableNameConcurrently()
+      throws ExecutionException, InterruptedException {
+
+    ExecutorService pool = Executors.newFixedThreadPool(64);
+
+    for (int i = 0; i < 30; i++) {
+      String tablename = "t" + i;
```

Comment:
Could simplify with:
```suggestion
    for (String tablename : getUniqueNames(30)) {
```